main.go

```diff
@@ -258,6 +267,13 @@ func loadConfig(filename string, queryUserData bool) (*Config, error) {
 		UsersDir:                   "C:\\Users",
 		CleanUpTaskDirs:            true,
 		IdleShutdownTimeoutSecs:    0,
+		WorkerTypeMetaData: map[string]interface{}{
```

Comment:
We allow any json object for the worker type meta data, so using map[string]interface{} guarantees that regardless of what a worker type creator puts in the worker type definition under this new key, it can be stored, merged with the default set of keys, and then marshalled back to json again.